{"name": "test_hello2", "status": "failed", "statusDetails": {"message": "AssertionError: assert 0", "trace": "self = <scripts.test_hello.TestHello object at 0x000001BBF8DABDA0>\n\n    def test_hello2(self):\n    \n    \n>       assert 0\nE       assert 0\n\nscripts\\test_hello.py:15: AssertionError"}, "start": 1583394491819, "stop": 1583394491819, "uuid": "daf466d9-295a-4ca0-ac77-9f5547aea755", "historyId": "1bf1c3dbd7d578643bdc2762affb01b7", "testCaseId": "2d4f9395c65ae8ee4318420c36e6960c", "fullName": "scripts.test_hello.TestHello#test_hello2", "labels": [{"name": "parentSuite", "value": "scripts"}, {"name": "suite", "value": "test_hello"}, {"name": "subSuite", "value": "TestHello"}, {"name": "host", "value": "DESKTOP-PTRS48T"}, {"name": "thread", "value": "5076-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "scripts.test_hello"}]}